.all-contributorsrc

```diff
@@ -453,6 +453,34 @@
       "contributions": [
         "doc"
       ]
+    },
```

Comment:
uber-nit: maybe merge (or reset?) upstream - this file should not appear in git diff IMHO